doc/source/regression/dwt-idwt.md

```diff
@@ -75,7 +52,7 @@ Voilà! That's it!
 
 ## More examples
 
-Now, let's experiment with `dwt` some more. For example, let's pass a
+Now let's experiment with `dwt()` some more. For example, let's pass a
```

Comment:
@agriyakhetarpal, this is also a stylistic choice we can revert if you like, but I personally like empty parentheses appended to the end of function names because I find it helpful to sort names into those that can be read versus those that must be called. 

This is also how it was prior to your changes.

Reply:
Ah, yes, we should revert it in that case – I don't remember why I removed the parentheses.